backend/app/neo4j_driver.py

```diff
@@ -146,7 +146,187 @@ def get_mismatched_authors_committers(self):
                     "authored_date": r["authored_date"]
                 }
                 for r in result
-            ]
+                ]
+
+    def merge_organization(self, name: str, slug: str = None):
+        slug = slug or name.lower().replace(" ", "-")
+        with self.driver.session() as session:
+            result = session.run(
+                """
+                MERGE (o:Organization {slug: $slug})
+                ON CREATE SET o.name = $name
```

Comment:
```suggestion
                ON CREATE SET o.name = $name, o.uuid = randomUUID()
```

[docs](https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-randomuuid)

Reply:
i made it take it from arguments so we can import flat files that reference it in the future

Reply:
??